cr.swift

```diff
@@ -0,0 +1,41 @@
+//
+//  Netguru iOS code review task
+//
+
+class paymentViewController: UIViewController {
```

Comment:
```suggestion
final class PaymentViewController: UIViewController {
```
Declare final to avoid unexpected overrides and increase of compile time